src/features/amUI/systemUser/components/DelegationCheckError/DelegationCheckError.tsx

```
@@ -24,14 +27,131 @@ export const DelegationCheckError = ({
+const ResourceReasonDetails = ({ codes }: ResourceReasonDetailsProps) => {
+  const { t } = useTranslation();
+
+  if (Array.isArray(codes)) {
+    if (codes.length > 1) {
+      return (
+        <DsListUnordered>
+          {codes.map((code) => {
+            const error = mapErrorCodeToErrorMessage(
+              ReasonErrorMap[code] || ReasonErrorMap.Unknown,
+            );
+            return <DsListItem>{t(error) || ''}</DsListItem>;
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)